apps/site/assets/.stylelintrc

```diff
@@ -39,6 +39,7 @@
       2,
       {
         "ignore": [
+          "param",
```

Comment:
What does this do?

Reply:
https://stylelint.io/user-guide/rules/list/indentation/#ignore-inside-parens-param-value

Allows arbitrary indentation of function params. 
TLDR Lets me write this without complaining haha

```scss
@mixin agenda-bullet($bg-color: $gray, $color: $white,
                     $counter, $counter-type: false)
```